RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/card-list.lava

```diff
@@ -19,14 +17,18 @@
             {% assign cciid = item.Id %}
     	    {% assign guid = item.Guid %}
     	    {% assign channelNameParts = item.ContentChannel.Name | Split:' - ' %}
-    	    {% assign type = channelNameParts[1] | Singularize %}
+			{% if item.ContentChannelId == 26 %}
+				{% capture type %}{{ item | Attribute:'Category' }} {{ channelNameParts[1] | Singularize }}{% endcapture %}
+			{% else %}
+    	    	{% assign type = channelNameParts[1] | Singularize %}
+			{% endif %}
     	    {% assign title = item.Title | Replace:"'","’" %}
             {% assign titlesize = "h4" %}
 			{% assign dates = item | Attribute:'Dates','RawValue' %}
 			{% assign dateParts = dates | Split:',' %}
             {% capture subtitle %}
 				{%- comment -%}Check this item's channel to see if dates should be visible{%- endcomment -%}
-				{% if showDate == 'Yes' %}
+				{% if showdates != 'false' showDate == 'Yes' %}
```

Comment:
I'm not sure if this is working. You've got 2 conditions and no `&&` or `||` here.

Reply:
Great catch! Thank you!